manta/client.py

```diff
@@ -195,9 +199,13 @@ def _request(self, path, method="GET", query=None, body=None, headers=None):
                 headers["Date"] = http_date()
             sigstr = 'date: ' + headers["Date"]
             algorithm, fingerprint, signature = self.signer.sign(sigstr)
-            headers["Authorization"] = \
-                'Signature keyId="/%s/keys/%s",algorithm="%s",signature="%s"' % (
-                    self.account, fingerprint, algorithm, signature)
+            auth = ('Signature keyId="/{}/keys/{}",algorithm="{}",signature="{}"'
+                    .format('/'.join(filter(None, [self.account, self.subuser])),
+                            fingerprint, algorithm, signature))
+            headers["Authorization"] = auth
```

Comment:
AFAIK we can't use `str.format()`, at least not with empty fields, and still support Python 2.6:

```
$ python2.6
Python 2.6.9 (unknown, Oct 23 2015, 18:05:10)
[GCC 4.2.1 Compatible Apple LLVM 7.0.0 (clang-700.0.59.5)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> template = 'Signature keyId="/{}/keys/{}",algorithm="{}",signature="{}"'
>>> template.format('a', 'b', 'c', 'd')
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ValueError: zero length field name in format
```

vs. python 2.7:

```
$ python2.7
Python 2.7.10 (default, Oct 23 2015, 18:05:06)
[GCC 4.2.1 Compatible Apple LLVM 7.0.0 (clang-700.0.59.5)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> template = 'Signature keyId="/{}/keys/{}",algorithm="{}",signature="{}"'
>>> template.format('a', 'b', 'c', 'd')
'Signature keyId="/a/keys/b",algorithm="c",signature="d"'
```

TBH I'm pretty ignorant on `str.format()`, being a Python 1.5.2 cro-magnon man.